tests/playwright/pages/wp-admin-page.js

```diff
@@ -105,4 +105,28 @@ module.exports = class WpAdminPage extends BasePage {
 		await this.page.selectOption( '#WPLANG', language );
 		await this.page.locator( '#submit' ).click();
 	}
+
+	async gotoThemesPage() {
+        await this.page.goto( '/wp-admin/themes.php', { waitUntil: 'networkidle' } );
+    }
+
+	async activateHelloTheme() {
+        await this.gotoThemesPage();
+		if ( await this.page.locator( '[aria-label="Activate Hello Elementor"]' ).isVisible() ) {
```

Comment:
Is it possible to use a classname or ID for this, to make it language agnostic?

Reply:
This button only contains an aria-label with the theme name. I could not find a way to create a selector where you just change out the ID or class name. So now I made it one function and you can pass in any selector to activate it.